src/desktop/apps/auction/components/layout/auction_info/__tests__/Registration.jest.js

```diff
@@ -3,7 +3,7 @@ import { test } from "desktop/apps/auction/components/layout/auction_info/Regist
 
 const { Registration } = test
 
-xdescribe("auction/components/layout/auction_info/Registration.test", () => {
+describe("auction/components/layout/auction_info/Registration.test", () => {
```

Comment:
@bhoggard bringing back these tests as part of this PR. Luckily most of them passed with little to no fixes.